src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -643,10 +641,7 @@ String^ UnitConverterViewModel::Serialize()
         String^ serializedData = ref new String(wstring(out.str()).c_str());
         return serializedData;
     }
-    else
-    {
-        return nullptr;
-    }
+    return nullptr;
```

Comment:
nit: The general coding style in the project is to have a line-feed after a closing statement (`}`). Please add one here.